CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2019.10.0
+* Enhancement - Add `MTU` field into ENI config. [#95](https://github.com/aws/amazon-ecs-cni-plugins/pull/95)
+
+## 2019.09.0
+* Enhancement - Add `stay-down` config to ENI plugin. [#94](https://github.com/aws/amazon-ecs-cni-plugins/pull/94)
```

Comment:
i think this entry should be moved to 2019.10.0 and 2019.09.0 should be removed

Reply:
+1